### PR TITLE
Add desktop version

### PR DESCRIPTION
**Pull request by Samuelwanza**
-Refactored the index html to fit both mobile and desktop versions
-Styled the desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,178 +67,188 @@ <h3>LET’S CONNECT</h3>
...
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li class="margin-spacer">CSS</li>
+            <li class="margin-spacer">JavaScript</li>
+          </ul>
+          <button class="margin-spacer">see Project</button>
         </div>
+      </div>
```
Comment (MaganaAsiati) [inline index.html:191]: - Kindly make sure your card design stick as much as possible to the Figma design,  make sure that you reduce the space between the left and right sides of your card and add the height of your card and reduce the width to make it stick to the Figma. please check the comparison below :+1: 


## Figma design 

image

## You design 


image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @Samuelwanza 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 2]
```diff
@@ -57,178 +67,188 @@ <h3>LET’S CONNECT</h3>
...
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li class="margin-spacer">CSS</li>
+            <li class="margin-spacer">JavaScript</li>
+          </ul>
+          <button class="margin-spacer" type="button">See Project</button>
         </div>
```
Comment (wayungi) [inline index.html:165]: - [ ] Your project cards are too small for the layout. Kindly increase their size to match the Figma. This change was requested by the previous reviewer but you have reduced them way too much. Kindly take guidance from the figma
 **figam**
 
width1

**Your design**
width

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Samuelwanza , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- [ ] All linter checks passing ✔️ 
- [ ]  Correct GitHub flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 3]
```diff
@@ -371,3 +384,253 @@ textarea::placeholder {
...
+  }
+
+  .grid-item {
+    display: flex;
+    flex-direction: row;
+    width: 90%;
+    height: 300px;
+    justify-content: space-between;
+    background: white;
+  }
```
Comment (AbelG101) [inline index.css:492]: - [ ] Please make sure the changes you introduce are aligned with the Figma design as I've noticed you're project images are stretching too much. see below 👇
  image
- [ ] So in order to fix this issue you can start by removing the fixed heights you've set: 
    - [ ] 1st remove `height: 300px;` from the `.grid-item` class 
    - [ ] 2nd remove `height: 280px;` from the `.image-wrapper > img` selector at line `498`. 
    - [ ] Finally, you can make some other refinements in accordance 😉

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Samuelwanza 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- I'd advise you to double check the changes you make are in sync with the Figma design. You just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by NahnahAJ — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @Samuelwanza , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**